docs/features/dead_letter_archive.md

```diff
@@ -19,3 +19,80 @@ In addition to the above, as of Cumulus v9+, the Cumulus API also contains a new
+
+```ts
+{
+    body: [string], // parseable as EventBridge
+    error: [string | null], // error that caused the message to be shunted to the DLQ
+    execution: [string | null], // execution ARN for the execution which created the originating sf event
+    time: [string | null], // Zulu timestamp of of the originating sf event
+    collection: [string | null], // collection the granule belongs to
+    granules: [Array[string | null] | null], // granules 
+    stateMachine: [string | null], // ARN of the triggering workflow
+    status: [string | null], status of triggering execution
+    /* these following are standard, not built by cumulus */
+    md5OfBody: [string], // checksum of message body
+    eventSource: [string], // aws:sqs
+    awsRegion: [string], // aws region that this is happening in
+    messageId: [string], // uniqueID of the DLQ message
+    receiptHandle: [string], // An identifier associated with the act of receiving the message. A new receipt handle is returned every time you receive a message.
+    attributes: [Object], // A map of the attributes requested in ReceiveMessage to their respective values.
+    messageAttributes: [Object], // Each message attribute consists of a Name, Type, and Value. 
+}
+```
+
+note that each of these fields except for 'body' can be null if no data was found, usually due to a parsing error
+for further details on body contents: [see below]
+
+## Dead Letter Archive Body contents
+
+the body attribute should be a JSON string containing an event bridge event
+
+Note that
+
+- the body attribute *can* come nested, such that you will have to de-nest a series of body attributes to get to the heart of your message
+- the word body can be interchanged with Body (capitalized)
+- because this message body arrived in the Dead Letter Archive because of issues in processing it, there is no strict guarantee that it is a valid json object, or conforms to expected structure. the *expected* structure follows.
+
+```ts
+{
+    version: [string | null], // versionString
+    id: [string | null], // unique ID of the triggering event
+    'detail-type': 'Step Functions Execution Status Change', // defines the below 'detail' spec
+    source: 'aws.states',
+    account: [string], // account ID
+    time: [string], // Zulu timestamp of the originating sf event
+    region: [string], //aws region
+    resources: [Array[string]], //ARNs of involved resources
+    detail: [string], //parses as Step Function Execution Status Change object, see below
+}
+```
+
+Step Function Execution Status Change (detail) [here](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-client-sfn/Interface/DescribeExecutionCommandOutput/):
+
+```ts
+{
+    stateMachineArn: [string], // ARN of the triggering workflow
```

Comment:
so that's right, I missed it.
I do want to clarify, do youhave execution_arn or executionArn ? because I have camelcase everywhere I've looked and in the docs